osu.Game/Screens/OnlinePlay/Playlists/PlaylistsMatchSettingsOverlay.cs

```diff
@@ -26,8 +26,16 @@ public class PlaylistsMatchSettingsOverlay : MatchSettingsOverlay
     {
         public Action EditPlaylist;
 
+        private MatchSettings settings;
+
+        protected override OsuButton SubmitButton => settings.ApplyButton;
+
+        protected override bool IsLoading => settings.IsLoading; // should probably be replaced with an OngoingOperationTracker.
+
+        protected override void SelectBeatmap() => settings.SelectBeatmap();
```

Comment:
This is intentionally directed through the button (rather than calling `EditPlaylist` directly) to animate the button... at least in theory https://github.com/ppy/osu/issues/14132